examples/chart/teleport-relay/values.yaml

```diff
@@ -0,0 +1,606 @@
+# which can be optionally used to access a private Docker registry.
+#
+# See the [Kubernetes
+# reference](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod)
+# for more details.
+imagePullSecrets: []
+
+# imagePullPolicy(string) -- sets the pull policy for any pods created by the
+# chart. See [the Kubernetes
+# documentation](https://kubernetes.io/docs/concepts/containers/images/#updating-images)
+# for more details.
+imagePullPolicy: IfNotPresent
+
+# topologySpreadConstraints(list) -- sets the topology spread constraints for
+# any pods created by the chart. See [the Kubernetes
+# documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/)
+# for more details.
+#
+# When unset, the chart defaults to a soft topology spread constraint that tries
+# to spread pods across hosts and zones.
+#
+# ```
+# topologySpreadConstraints:
+#   - maxSkew: 1
+#     topologyKey: kubernetes.io/hostname
+#     whenUnsatisfiable: ScheduleAnyway
+#     labelSelector:
+#       matchLabels: # dynamically computed
+#   - maxSkew: 1
+#     topologyKey: topology.kubernetes.io/zone
+#     whenUnsatisfiable: ScheduleAnyway
+#     labelSelector:
+#       matchLabels: # dynamically computed
+# ```
+topologySpreadConstraints: []
+
+# affinity(object) -- sets the affinities for any pods created by the chart. See
+# [the Kubernetes
+# documentation](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
+# for more details.
+affinity: {}
+
+# tolerations(list) -- sets the tolerations for any pods created by the chart.
+# See [the Kubernetes
+# documentation](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+# for more details.
+tolerations: []
+
+# teleportConfig(object) -- contains YAML teleport configuration to pass to the
+# Teleport pods. The configuration will be merged with the chart-generated
+# configuration and will take precedence in case of conflict.
+#
+# See the [Teleport Configuration Reference](../../reference/deployment/config.mdx) for the list of supported fields.
+teleportConfig: {}
+
+# extraArgs(list) -- contains extra arguments to pass to `teleport start` for
+# the main Teleport container.
+extraArgs: []
+
+# extraEnv(list) -- contains extra environment variables to set in the main
+# Teleport container.
+#
+# For example:
+# ```yaml
+# extraEnv:
+#   - name: HTTPS_PROXY
+#     value: "http://username:password@my.proxy.host:3128"
+# ```
+extraEnv: []
+
+# extraContainers(list) -- contains extra containers to add in the main Teleport
+# pod.
+#
+# For example:
+# ```yaml
+# extraContainers:
+# - name: debug-sidecar
+#   command:
+#     - busybox
+#     - sh
+#     - -c
+#     - "echo waiting && sleep infinity"
+#   image: busybox:latest
+#   imagePullPolicy: IfNotPresent
+#   securityContext:
+#     privileged: true
+#     runAsNonRoot: false
+# ```
+extraContainers: []
```

Comment:
I wonder if we should not add this by default and see if one day someone has a good use-case for this. While it can make sense for teleport-cluster (init) or agent (fetch secrets), I don't see why someone would add an extra container into the relay pod.